decidim/decidim
# Create user notifications while seeding

#### :tophat: What? Why?
Currently, Decidim is not adding user notifications while seeding, making it hard for users to actually see the notication page functionality.

This PR adds the Comments, Proposals and Meetings created in the Participatory Processes to the notifications list. 

#### Testing
1. Reseed your db or create a new dev app 
2. Boot the application 
3. Login as admin 
4. Visit notifications page 
5. See there are no notifications 
6. Apply Patch 
7. Repeat steps 1,2,3 and 4
8. See there are notifications

:hearts: Thank you!
